doc/classes/Variant.xml

```diff
@@ -1,49 +1,63 @@
-		# bar = "Uh oh! I can't make statically typed variables become a different type!"
+		# bar = "Uh oh! I can't change the type of static variables!"
 		[/gdscript]
 		[csharp]
-		// C# is statically typed. Once a variable has a type it cannot be changed. You can use the `var` keyword to let the compiler infer the type automatically.
-		var foo = 2; // Foo is a 32-bit integer (int). Be cautious, integers in GDScript are 64-bit and the direct C# equivalent is `long`.
+		// C# is statically typed. Once a variable has a type it cannot be changed.
+		// You can use the `var` keyword to let the compiler infer the type automatically.
+		var foo = 2; // Foo is a 32-bit integer (int).
 		// foo = "foo was and will always be an integer. It cannot be turned into a string!";
-		var boo = "Boo is a string!";
+		var boo = "boo is a string!";
 		var ref = new RefCounted(); // var is especially useful when used together with a constructor.
 
 		// Godot also provides a Variant type that works like a union of all the Variant-compatible types.
 		Variant fooVar = 2; // fooVar is dynamically an integer (stored as a `long` in the Variant type).
 		fooVar = "Now fooVar is a string!";
-		fooVar = new RefCounted(); // fooVar is a GodotObject.
+		fooVar = new RefCounted(); // fooVar is now a GodotObject.
 		[/csharp]
 		[/codeblocks]
-		Godot tracks all scripting API variables within Variants. Without even realizing it, you use Variants all the time. When a particular language enforces its own rules for keeping data typed, then that language is applying its own custom logic over the base Variant scripting API.
-		- GDScript automatically wrap values in them. It keeps all data in plain Variants by default and then optionally enforces custom static typing rules on variable types.
-		- C# is statically typed, but uses its own implementation of the Variant type in place of Godot's [Variant] class when it needs to represent a dynamic value. C# Variant can be assigned any compatible type implicitly but converting requires an explicit cast.
-		The global [method @GlobalScope.typeof] function returns the enumerated value of the Variant type stored in the current variable (see [enum Variant.Type]).
+
+		Godot tracks all scripting API variables within Variants. When a particular language enforces its own rules for keeping data typed, that language is applying its own custom logic over the base Variant scripting API:
+		- GDScript automatically wraps values in them. It keeps all data in plain Variants by default and then optionally enforces custom static typing rules on variable types.
+		- C# is statically typed, but uses its own implementation of Variant when it needs to represent a dynamic value. A [Variant] can be assigned any compatible type implicitly, but converting requires an explicit cast.
+
+		A Variant in Godot takes up only 20 bytes and can store almost any datatype inside of it. Those can be roughly categorized as:
```

Comment:
I think this should be 24 bytes instead of 20.  There's also a comment in variant.h with more details about cases where it could be larger when using doubles or allocate more, not sure how much of this makes sense to include:

```
	// Variant takes 24 bytes when real_t is float, and 40 bytes if double.
	// It only allocates extra memory for AABB/Transform2D (24, 48 if double),
	// Basis/Transform3D (48, 96 if double), Projection (64, 128 if double),
	// and PackedArray/Array/Dictionary (platform-dependent).
```

Reply:
Yeah, again. I remembered there was a comment about it somewhere. The docs have been hilariously wrong about it for so long, huh? Or what if this comment is technically incorrect by just a few bytes?

Reply:
Yeah this remark never made sense to me. I think we should just remove it and not mention it, since it seems dependent on the variant.